package.json

```diff
@@ -2,6 +2,17 @@
   "name": "front-robert",
   "version": "0.1.0",
   "private": true,
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "eslint --fix",
```

Comment:
Do we need to call prettier - - write too?

Reply:
I think this would be nice too
The thing is that I'd prefer that everybody has a well configured IDE too.
But i'll add this too yep ;)